Source/WebInspectorUI/UserInterface/Views/ColorPicker.js

```diff
@@ -89,6 +89,25 @@ WI.ColorPicker = class ColorPicker extends WI.Object
 
         this._color = WI.Color.fromString("white");
 
+        if (colorPalette) {
+            let colorSwatchDisplay = this._element.appendChild(document.createElement("div"));
+            colorSwatchDisplay.classList.add("color-swatch-display");
+
+            let swatchesTitle = colorSwatchDisplay.appendChild(document.createElement("h2"));
+            swatchesTitle.innerText = WI.UIString("Variables", "Variables", "Title of swatches section in Color Picker");
+            
+            let swatchWrapper = colorSwatchDisplay.appendChild(document.createElement("ul"));
+            this._sortedPalette = this._specializedSort(colorPalette);
+
+            for (let paletteObject of this._sortedPalette) {
+                let swatch = new WI.InlineSwatch(WI.InlineSwatch.Type.Color, paletteObject.color, {readOnly: true, tooltip: paletteObject.colorSource});
+                swatch.element.addEventListener("click", (event) => {
+                    this._colorSwatchDispalyColorSelected(paletteObject.color, paletteObject.colorSource);
+                });
+                swatchWrapper.appendChild(document.createElement("li").appendChild(swatch.element));
```

Comment:
this is not doing what you want it to do.  `Node.prototype.appendChild` returns the provided child, not the parent element.  as such, i think you meant to do this
```
swatchWrapper.appendChild(document.createElement("li")).appendChild(swatch.element);
```